annotations/forms.py

```diff
@@ -27,10 +27,14 @@ class RegistrationForm(forms.Form):
 	"""
 	Gives user form of signup and validates it.
 	"""
+	full_name = forms.CharField(widget=forms.TextInput(attrs=dict(required=True, max_length=30, render_value=False)), label=_("Full name"))
```

Comment:
Yikes, this adds a whole lot of code! Do you really need to explicitly set the widgets on all of these fields? The defaults are usually enough. If you're worried about labels, there are other (easier) places to change those.

Also, don't let your lines exceed 79 characters. ;-)